interface-definitions/service_ipoe-server.xml.in

```diff
@@ -174,10 +174,28 @@
                   </leafNode>
                 </children>
               </node>
+              <leafNode name="lua-username">
+                <properties>
+                  <help>Username function</help>
+                  <valueHelp>
+                    <format>txt</format>
+                    <description>Name of function in lua file to construct username</description>
+                  </valueHelp>
+                </properties>
+              </leafNode>
               #include <include/accel-ppp/vlan.xml.i>
               #include <include/accel-ppp/vlan-mon.xml.i>
             </children>
           </tagNode>
+          <leafNode name="lua-file">
```

Comment:
We should use the `file-path` validator with `--parent-dir` to verify that the script actually exists and it's inside `/config/scripts` so that it's properly migrated to new images on upgrades.